src/uu/wc/src/wc.rs

```diff
@@ -808,6 +810,12 @@ fn files0_iter<'a>(
     })
 }
 
+fn escape_name_wrapper(name: &OsStr) -> String {
+    escape_name(name, QS_ESCAPE)
```

Comment:
Same as in `ls`, `use uucore::quoting_style` and `quoting_style::escape_name`.

Reply:
Done.